src/clusterfuzz/_internal/issue_management/google_issue_tracker/client.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Gets a Google Issue Tracker HTTP client."""
+
+import os
+import urllib
+
+import google.oauth2.credentials
+import google_auth_httplib2
+import googleapiclient
+import httplib2
+
+_ROLE_ACCOUNT = "cluster-fuzz-google-issue-tracker"
```

Comment:
nit: can you use single quotes for all strings?

Reply:
Thanks, looks like my linter was auto-"correcting" this.